tdrs-backend/tdpservice/users/admin.py

```diff
@@ -60,6 +36,7 @@ class UserAdmin(admin.ModelAdmin):
     readonly_fields = ['last_login', 'date_joined', 'login_gov_uuid', 'hhs_id', 'access_request', 'deactivated']
     form = UserForm
     list_filter = ('account_approval_status',)
+    autocomplete_fields = ['stt']
```

Comment:
I thought this would be a nice add - it makes the `stt` dropdown select in the django admin searchable by the `STTAdmin.search_fields` values (`name` and `stt_code`).